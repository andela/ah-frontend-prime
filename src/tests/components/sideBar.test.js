import React from "react";
import { shallow } from "enzyme";
import SideBar from "../../components/sideDrawer";

let wrapper;
let instance;
describe("test for sidebar  component", () => {
  beforeEach(() => {
    wrapper = shallow(<SideBar />);
  });
  it("should render without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
