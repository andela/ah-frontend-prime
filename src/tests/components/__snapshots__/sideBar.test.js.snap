// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for sidebar  component should render without crashing 1`] = `ShallowWrapper {}`;
